TGC/events/Germany VS Hungary Events.txt

```diff
@@ -1795,7 +1795,7 @@
 		}
 	}
 
-## Siebenb�rgen conference - Embarresment ##
+## Siebenb?rgen conference - Embarresment ##
```

Comment:
Possible encoding error, please take a look at it.

Reply:
Just a missing character in the comment, nothing to worry about. But I fixed it anyway